src/platform/native.zig

```diff
@@ -214,6 +223,45 @@ pub const Platform = struct {
         while (platform.events.popFirst()) |ev| {
             platform.allocator.destroy(ev);
         }
+
+        platform.deinitLinuxGamemode();
+    }
+
+    fn initLinuxGamemode(allocator: std.mem.Allocator) error{ OutOfMemory, InvalidUtf8 }!void {
+        if (builtin.os.tag == .linux) {
+            const gamemode = @import("gamemode");
+            const GAMEMODE_ENV = try getEnvVarOwned(allocator, "GAMEMODE");
+            if (GAMEMODE_ENV == null) {
+                gamemode.requestStart() catch |err| {
+                    std.log.err("Gamemode error {} -> {s}", .{ err, gamemode.errorString() });
+                };
+            } else {
+                if (!std.ascii.eqlIgnoreCase(GAMEMODE_ENV.?, "off")) {
```

Comment:
Maybe check for both `off` and `false` here?

Reply:
We could store whether or not we used gamemode in a field of `platform` struct, so that we can reuse it simply in the deinit code without duplicating the env var handling logic